### ISSUE TITLE
Rename D0Tohh to D0ToKpi

### ISSUE THREAD

**Issue by alexpearce** [alexpearce/CharmProduction#73]
Now stuff is more stable, it makes sense to me that `D0Tohh` should be renamed to `D0ToKpi`. I know the stripping lines captures all h+h- decays, but we really don't care about anything other than [K- pi+]CC, and all the other `CharmProductionNtuple` instances are mode specific, so it would make things more consistent.

Refactoring would just be a search and replace, and I suppose we could even have [`decaytreetuples.py`](https://github.com/alexpearce/CharmProduction/blob/89a2570b450fd8384248d5a8be319efbf0e0741d/options/decaytreetuples.py) only capture the K-pi+ decay as well.

Thoughts?

**Comment by chrisburr** [comment]
Definitely, this will make things a lot easier. For our new stripping line does it make sense to do this as well?

Also am I correct in thinking this will still catch D0 -> K+ pi-? I guess it just becomes a, negligible(?), systematic?

**Comment by alexpearce** [comment]
We will probably keep the code “for posterity” but disable the non-K⁻pi⁺ lines (negative prescale). 2010 assumed that the doubly Cabibbo-suppressed yields were negligible in comparison with the Cabibbo-favoured — the branching fraction is around 200 times smaller — but if we get much larger yields it will certainly be something to think about. I agree that if it was assigned as a systematic it'd be small, especially compared with other things.
